src/components/Hoverable/index.native.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * On mobile, there is no concept of hovering, so we return a plain wrapper around the component's only child.
+ */
+import React from 'react';
+import {propTypes, defaultProps} from './HoverablePropTypes';
+
+const Hoverable = props => React.Children.only(props.children);
```

Comment:
Not sure everyone will get what `React.Children.only` does. If you want to leave a note somewhere that would be cool.